core/src/main/java/com/predic8/membrane/core/interceptor/apikey/ApiKeysInterceptor.java

```diff
@@ -33,6 +33,10 @@
 import static com.predic8.membrane.core.interceptor.Outcome.RETURN;
 import static java.util.stream.Stream.ofNullable;
 
+/**
```

Comment:
Instead of proxies use APIs
Can this be rewritten in a more clearer way easier to read for beginners